### PR TITLE
Review/update spec for NotEqual operation

### PR DESCRIPTION
**Pull request by nsemaev**
### Details:
 - *Restoring the correct specification structure*

### Tickets:
 - *45998, 37459*

### CODE REVIEW

File: docs/ops/comparison/NotEqual_1.md [hunk 1]
```diff
@@ -4,7 +4,18 @@
 
 **Category**: Comparison binary operation
 
-**Short description**: *NotEqual* performs element-wise comparison operation with two given tensors applying multi-directional broadcast rules.
+**Short description**: *NotEqual* performs element-wise comparison binary operation with two given tensors applying
+multi-directional broadcast rules.
```
Comment (rkazants) [inline docs/ops/comparison/NotEqual_1.md:8]: `binary` is extra. We don't use this word for similar operation description. Please remove it

File: docs/ops/comparison/NotEqual_1.md [hunk 2]
```diff
@@ -4,7 +4,18 @@
...
+
+**Detailed description**
+Before performing comparison operation, input tensors *a* and *b* are broadcasted if their shapes are different and
+`auto_broadcast` attributes is not `none`. Broadcasting is performed according to `auto_broadcast` value.
```
Comment (rkazants) [inline docs/ops/comparison/NotEqual_1.md:12]: please re-write it to:
`Before performing comparison operation, input tensors *a* and *b* are broadcasted if their shapes are different. Broadcasting is performed according to `auto_broadcast` value.`

File: docs/ops/comparison/NotEqual_1.md [hunk 3]
```diff
@@ -13,7 +24,8 @@
   * **Description**: specifies rules used for auto-broadcasting of input tensors.
   * **Range of values**:
     * *none* - no auto-broadcasting is allowed, all input shapes should match
-    * *numpy* - numpy broadcasting rules, aligned with ONNX Broadcasting. Description is available in <a href="https://github.com/onnx/onnx/blob/master/docs/Broadcasting.md">ONNX docs</a>.
+    * *numpy* - numpy broadcasting rules, aligned with ONNX Broadcasting. Description is available in
+      <a href="https://github.com/onnx/onnx/blob/master/docs/Broadcasting.md">ONNX docs</a>.
```
Comment (rkazants) [inline docs/ops/comparison/NotEqual_1.md:28]: also pdpd mode is missed. Please make sure that implementation in nGraph reference contains it

File: docs/ops/comparison/NotEqual_1.md [hunk 4]
```diff
@@ -4,7 +4,18 @@
...
 
-**Short description**: *NotEqual* performs element-wise comparison operation with two given tensors applying multi-directional broadcast rules.
+**Short description**: *NotEqual* performs element-wise comparison operation with two given tensors applying
+multi-directional broadcast rules.
+
+**Detailed description**
+Before performing comparison operation, input tensors *a* and *b* are broadcasted if their shapes are different.
+Broadcasting is performed according to `auto_broadcast` value.
+
+After broadcasting *NotEqual* does the following with the input tensors *a* and *b*:
```
Comment (tsavina) [inline docs/ops/comparison/NotEqual_1.md:14]:
```suggestion
After broadcasting, *NotEqual* does the following with the input tensors *a* and *b*:
```

**Review by tsavina — APPROVED** [review]

**Review by rkazants — APPROVED** [review]

**Pull request merged**